man/systemd.network.xml

```diff
@@ -700,6 +700,17 @@
           <literal>[IPv6PrefixDelegation]</literal> and the <literal>[IPv6Prefix]</literal>
           sections for more configuration options.
           </para></listitem>
+	</varlistentry>
+        <varlistentry>
+          <term><varname>IPv6PDSubnetId=</varname></term>
+          <listitem><para>Configure a specific subnet id from a received prefix delegation to
+          the interface. Allowed values are values from the range 0 to 9223372036854775808
+          (0x8000000000000000, 63 bit) to select a subnet id (as defined in
```

Comment:
you probably want to say `2^63` here, rather than `63 bit`, no?

But this is a bit confusuing, you say the range starts at 0, which suggests the range actually ends at 2^63-1, i.e. 9223372036854775807, aka 0x7FFFFFFFFFFFFFFF), no?

Reply:
we usually capitalize id as ID btw